### DISCUSSION TITLE
ImportError: localtileserver is not installed. Please install it before proceeding google collab

### DISCUSSION THREAD

**Discussion by johngeoscrub** [opengeos/segment-geospatial#219] [Q&A] [answered]
When running the example notebook[ input_prompts.ipynb](https://colab.research.google.com/github/opengeos/segment-geospatial/blob/main/docs/examples/input_prompts.ipynb) there is an issue where if we try load our exported image into our `leafmap `we get an import error that `localtileserver `is not installed - when installing it it stills throws the error so I imagine this must be a limitation in Google Collab? but just want to confirm it here

**Comment by giswqs** [comment, marked as answer]
Make sure you restart the runtime after installation

**Reply by johngeoscrub** [reply]
brilliant thanks for the responsiveness this is a fantastic package - great work!
